franklin/github/views.py

```diff
@@ -94,7 +95,7 @@ def repository_detail(request, pk):
         logger.warn(message + ' | %s | %s', request.user, site)
         return Response(message, status=status.HTTP_403_FORBIDDEN)
     if request.method == 'GET':
-        site_serializer = SiteSerializer(site)
+        site_serializer = SiteSerializer(site, context={'user': request.user})
```

Comment:
above comment is partly because we now call github for default environments that do not have a current deploy. Here we pass the current user into the context of the serializer so it can grab the oauth token for calling github.